src/env-inl.h

```diff
@@ -218,9 +218,11 @@ inline T* Environment::GetBindingData(v8::Local<v8::Context> context) {
       context->GetAlignedPointerFromEmbedderData(
           ContextEmbedderIndex::kBindingListIndex));
   DCHECK_NOT_NULL(map);
-  auto it = map->find(T::type_name);
-  if (UNLIKELY(it == map->end())) return nullptr;
-  T* result = static_cast<T*>(it->second.get());
+  constexpr size_t binding_index = static_cast<size_t>(T::binding_type_int);
+  static_assert(binding_index < std::tuple_size_v<BindingDataStore>);
+  auto ptr = (*map)[binding_index];
+  if (UNLIKELY(!ptr)) return nullptr;
```

Comment:
Is there any performance impact for keeping this `UNLIKELY`?

Reply:
it's on a hot path so I guess yes, anything that helps counts.